app/events/page.js

```diff
@@ -0,0 +1,235 @@
+import Navbar from "../components/navbar";
+import Image from "next/image";
+
+export default function Events() {
+  return (
+    <div className="bg-white">
+      <Navbar />
+      <div className="mx-auto max-w-2xl px-4 py-16 sm:px-6 sm:py-24 lg:max-w-7xl lg:px-8">
+        <h1 className="text-3xl font-bold tracking-tight text-black hr b-2 mt-8 flex-shrink-0 border-orange-500 border-b-4 rounded">
+          Upcoming events
+        </h1>
+
+        <div className="mt-6 grid grid-cols-1 gap-x-6 gap-y-10 sm:grid-cols-2 lg:grid-cols-4 xl:gap-x-16">
+          <div className="group relative">
+            <div className="aspect-h-1 aspect-w-1 w-full overflow-hidden rounded-md bg-gray-200 lg:aspect-none group-hover:opacity-75 lg:h-80">
+              <Image
+                src="/event1.png"
+                width={1920}
+                height={1080}
+                alt="Front of men&#039;s Basic Tee in black."
+                className="h-full w-full object-cover object-center lg:h-full lg:w-full"
+              />
+            </div>
+            <div className="mt-4 flex justify-between">
+              <div>
+                <h3 className="text-sm text-gray-700">
```

Comment:
Para que el [texto no te ocupe muchas lineas](https://tailwindcss.com/docs/text-overflow#truncate)
```suggestion
                <h3 className="truncate text-sm text-gray-700">
```